### PR TITLE
make it look like it was before

### PR DESCRIPTION
**Pull request by niklas-wortmann**

### CODE REVIEW

File: jump-start-tutorial/exercise-6_using-routeparams.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ We need several things to get there:
   </div>
   ```
 3. Add a new route to `APPROUTES` with the path `contacts/:id` that points to `ContactsDetailComponent`
-4. Change the `<mat-item>` in `ContactsListComponent`'s view to `<a mat-item>` and use the `RouterLink` directive with a DSL configuration that points to `ContactsDetailComponent`
+4. Change the `<mat-item>` in `ContactsListComponent`'s view to `<a mat-list-item>` and use the `RouterLink` directive with a DSL configuration that points to `ContactsDetailComponent`
```
Comment (0x-r4bbit) [inline jump-start-tutorial/exercise-6_using-routeparams.md]: Thanks @JWO719 ! The first one needs to be `mat-list-item` as well, would you mind updating your PR?

Comment (niklas-wortmann) [reply jump-start-tutorial/exercise-6_using-routeparams.md]: didn't get exactly what you mean?

Comment (0x-r4bbit) [reply jump-start-tutorial/exercise-6_using-routeparams.md]: > Change the `<mat-item>`

should be

> Change the `<mat-list-item>`

Comment (niklas-wortmann) [reply jump-start-tutorial/exercise-6_using-routeparams.md]: but the sentence is correctly? I mean it just says you should change the <mat-item> like it is in the current state of development to...

I'm a bit confused to be honest....

Comment (0x-r4bbit) [reply jump-start-tutorial/exercise-6_using-routeparams.md]: So the point here is that previously the list item markup looked like this:

```
 <mat-list-item *ngFor="let contact of contacts; trackBy:trackByContactId">
  ...
 </mat-list-item>
```
Note that it's a `mat-list-item` not a `mat-item` (this is a broken tag).

So in other words: you fixed a bug in our description (thanks for that!) but it turns out there's another bug :D it shouldn't have been `mat-item` in the first place but `mat-list-item`.

Does that help?

**Comment by niklas-wortmann** [comment]
get it :D

**Comment by 0x-r4bbit** [comment]
Wonderful :D Let me know when this PR is updated and I'll merge it right away. Thanks for your contribution!

**Comment by niklas-wortmann** [comment]
it is updated. Hope to chat with you at angular connect!!

**Comment by 0x-r4bbit** [comment]
Unfortunately I'm not there this year cuz I'm in the middle of a move.. Have fun tho!

Merging :)

**Pull request merged**